ci-operator/step-registry/openshift/upgrade/gcp/loki/openshift-upgrade-gcp-loki-workflow.yaml

```diff
@@ -0,0 +1,20 @@
+workflow:
+  as: openshift-upgrade-gcp-loki
+  steps:
+    pre:
+    - chain: ipi-gcp-pre
+    - ref: ipi-install-loki
+    test:
+    - ref: openshift-e2e-test
+    post:
+    - ref: gather-loki
+    - chain: ipi-gcp-post
+    dependencies:
+      OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: "release:initial"
+      OPENSHIFT_UPGRADE_RELEASE_IMAGE_OVERRIDE: "${RELEASE_IMAGE_LATEST}"
```

Comment:
This should read 

```
       OPENSHIFT_UPGRADE_RELEASE_IMAGE_OVERRIDE: "release:latest"
```

Reply:
`dependencies` is a mapping from the _thing_ you depend on, in our parlance -- `release:latest` -- to the env in which we expose that

Reply:
Thanks, fixed in 743d632f1debe56f9f724b4f9d0e04eb8519fdf4